### ISSUE TITLE
Compilation warnings on Elixir 1.11

### ISSUE THREAD

**Issue by crbelaus** [schrockwell/bodyguard#70]
Compiling bodyguard on Elixir 1.11 throws the following warning:

```
==> bodyguard
Compiling 9 files (.ex)
warning: function actions/1 required by protocol Plug.Exception is not implemented (in module Plug.Exception.Bodyguard.NotAuthorizedError)
  lib/bodyguard/not_authorized_error.ex:8: Plug.Exception.Bodyguard.NotAuthorizedError (module)

Generated bodyguard app
```

**Comment by benkimpel** [comment]
#72 should resolve this

**Comment by schrockwell** [comment]
Fixed in v2.4.1
